src/__tests__/utils/puzzles.test.ts

```diff
@@ -0,0 +1,29 @@
+import { checkPuzzleAnswer } from '../../utils/puzzles';
+import { Puzzle } from '@prisma/client';
+
+it('should compare the user submitted puzzle answer with the ans key in the puzzle (from the db)', () => {
+  const puzzle = {
```

Comment:
As discussed in the meeting, move object definitions like this into some kind of `/mocks` directory